README.md

```diff
@@ -1,3 +1,37 @@
 # Grunt incuna plugins
 
 A set of useful build and testing plugins easily reusable across projects.
+
+* [Adding new plugins](#adding-new-plugins)
+* [Plugins](#plugins)
+ * [update-po-files](#update-po-files)
+
+## Plugins
+
+### `update-po-files`
+
+A plugin for automatically updating `.po` files in a given directory from a `template.pot` file in that same directory.
+Usage:
+```js
+'update-po-files': {
+    all: {
+        src: 'my-i18n-directory'
+    }
+},
+```
+
+Requires binary `msgmerge` to run, will complain if one isn't found. Because OS X comes with it's own version of `gettext`, but without the binary tools it is required to install and force link it via `brew`:
+```
+brew install gettext
+brew link gettext --force
+```
+
+## Adding new plugins
```

Comment:
It could be useful to nest this under a `#development` section and have the plugins under `# Using the plugins`.
I think it can be handy for people who aren't familiar with the project to have a section on just using/installing the grunt plugins for their project, and then another section for those who are going to do dev work on the plugins themselves.